jib-maven-plugin/CHANGELOG.md

```diff
@@ -11,6 +11,8 @@ All notable changes to this project will be documented in this file.
 
 ### Fixed
 
+- Fixed an issue where decyrpting Maven settings `settings.xml` wholesale causes the build to fail. We now decrypted only the parts that are required. ([#1709](https://github.com/GoogleContainerTools/jib/issues/1709))
```

Comment:
Sorry, should be "wholesale caused the build to fail" instead of "causes", and "We now decrypt" not "We now decrypted"

Reply:
:+1: yeah I got lazy